### PR TITLE
Improves mappers

### PR DESCRIPTION
**Pull request by alokmenghrajani**

### CODE REVIEW

File: server/src/main/java/keywhiz/service/daos/GroupDAO.java [hunk 1]
```diff
@@ -55,17 +55,17 @@ public void deleteGroup(Group group) {
...
   public Optional<Group> getGroupById(long id) {
     GroupsRecord r = dslContext.fetchOne(GROUPS, GROUPS.ID.eq(Math.toIntExact(id)));
     return Optional.ofNullable(r).map(
-        (rec) -> rec.map(new GroupMapper()));
+        rec -> new GroupMapper().map(rec));
   }
 
   public ImmutableSet<Group> getGroups() {
-    List<Group> r = dslContext.select().from(GROUPS).fetch().map(new GroupMapper());
+    List<Group> r = dslContext.selectFrom(GROUPS).fetch().map(new GroupMapper());
```
Comment (sul3n3t) [inline server/src/main/java/keywhiz/service/daos/GroupDAO.java]: Could use `selectFrom(...)` in the other DAOs to be consistent.

Comment (sul3n3t) [inline server/src/main/java/keywhiz/service/daos/GroupDAO.java]: Nevermind, I see the problem

**Comment by sul3n3t** [comment]
:+1:

File: server/src/main/java/keywhiz/service/daos/GroupDAO.java (hunk 1 above)
Comment (alokmenghrajani) [inline server/src/main/java/keywhiz/service/daos/GroupDAO.java]: You can't use `.join()` when doing `.selectFrom()` but you can do `.fetchInto()` to get the same result.

**Pull request merged**